src/custom/state/swap/reducer.ts

```diff
@@ -0,0 +1,79 @@
+import { createReducer } from '@reduxjs/toolkit'
+import { parsedQueryString } from 'hooks/useParsedQueryString'
+
+import { Field, replaceSwapState, selectCurrency, setRecipient, switchCurrencies, typeInput } from 'state/swap/actions'
+import { queryParametersToSwapState } from 'state/swap/hooks'
+
+export interface SwapState {
```

Comment:
we normally would call this file reducerMod. We do that to signal is a copy paste of some other file.